pkg/cmd/release/deploy/deploy.go

```diff
@@ -82,6 +82,9 @@ const (
 
 	FlagUpdateVariables            = "update-variables"
 	FlagAliasUpdateVariablesLegacy = "updateVariables"
+
+	FlagDeploymentFreezeNames          = "deployment-freeze-names"
```

Comment:
this should probably be singular, each instance of the parameter is single freeze name, which can be specified multiple times.
```suggestion
	FlagDeploymentFreezeName          = "deployment-freeze-name"
```

Reply:
Yes, good catch